units/steelhive/steel_dreadnought.cfg

```diff
@@ -10,7 +10,7 @@
     experience=150
     level=3
     alignment=neutral
-    advances_to=null
+    advances_to=Steel Dominant
```

Comment:
Maybe better to not enable this unit in MP, better if it stays as a campaign thing with `[modify_unit_type]`